tests/test_engine.py

```diff
@@ -150,6 +169,19 @@ def test_nested_transaction(self):
         self.successResultOf(trx1.close())
         assert not conn.in_transaction()
 
+    def test_savepoints(self):
```

Comment:
A negative test case would also be good, but as you noted elsewhere attempting to start a save point outside a transaction doesn't fail on all databases, including SQLite.  So I can't really think of one...